src/data/accessibility.json

```diff
@@ -0,0 +1,55 @@
+{
+    "meta": {
+        "title": "Accessibility Statement by the Robert Koch Institute and the Federal Ministry of Health",
+        "description": ""
+    },
+    "title": "Accessibility Statement by the Robert Koch Institute and the Federal Ministry of Health",
+    "section-android-main": {
+        "textblock": [
+            "This accessibility statement applies from version 2.17 of the <b>Corona-Warn-App (Android)</b> of the Robert Koch Institute and Federal Ministry of Health.",
+            "As a public entity within the meaning of Guideline (EU) 2016/2102, we strive to make our websites and mobile applications accessible in accordance with the provisions of the German Act on Equal Opportunities for Persons with Disabilities (BGG) and the Accessible Information Technology Regulation (BITV 2.0) implementing Guideline (EU) 2016/2102."
+        ]
+    },
+    "section-ios-main": {
+        "textblock": [
+            "This accessibility statement applies from version 2.17 of the <b>Corona-Warn-App (iOS)</b> of the Robert Koch Institute and Federal Ministry of Health.",
```

Comment:
Same comment about use of "from" as in Android section.

Reply:
Changed to "starting with version 2.17" in https://github.com/corona-warn-app/cwa-website/pull/2508/commits/ee6eee639cde3fba05e740980cdf6e86f0adb402, thanks for the suggestion!